documentation/modules/metrics/kafka-bridge/proc_kafka-bridge-enabling-grafana.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// metrics/assembly_metrics-kafka-bridge.adoc
+
+[id='proc-kafka-bridge-enabling-{context}']
+
+= Enabling the Kafka Bridge Grafana dashboard
```

Comment:
Similar to the above ... do we really need this procedure? Can't we just keep the list of items from the dashabord where you describe the location and remove this?

Reply:
Same thought as before. Isn't it even enforced by PR #3269 with the refactoring made by Paul?